app/src/main/java/org/simple/clinic/scanid/IndiaNHIDInfoPayload.kt

```diff
@@ -29,4 +29,17 @@ data class IndiaNHIDInfoPayload(
 
     @Json(name = "address")
     val address: String
-)
+) {
+  fun fromPayload(): IndiaNHIDInfo {
```

Comment:
A more appropriate name would be `toPatientPrefillInfo` wouldn't it? We want to say that the payload object is being converted to whatever the local model is.